sphinx/events.py

```diff
@@ -115,7 +115,7 @@ def emit(self, name: str, *args: Any,
                 raise
             except Exception as exc:
                 raise ExtensionError(__("Handler %r for event %r threw an exception") %
-                                     (listener.handler, name)) from exc
+                                     (listener.handler, name), exc)
```

Comment:
I think it is better to show the exception in the message like "threw an exception: %r". What do you think?

Addition to this, `from exc` is still worthy. So adding `exc` as a new argument is better to me.

Reply:
`ExtensionError` has a custom `__str__` which prints `exc`, so I thought it was best to leave it to `__str__` to handle including `exc`:
https://github.com/sphinx-doc/sphinx/blob/03e85df6175258bd6d6ec24aaeed8fc74a335651/sphinx/errors.py#L57-L67

I'm not sure what you mean by the second bit - should I put back `from exc`?

Reply:
Thank you for your comment. Indeed, it is not needed.

>I'm not sure what you mean by the second bit - should I put back from exc?

Yes. I think both giving `exc` as an argument and passing it via `from` clause is better.

Reply:
👍 I've put back the `from exc`